util/test/demos/vk/vk_descriptor_index.cpp

```diff
@@ -291,13 +295,12 @@ void main()
         VK_STRUCTURE_TYPE_DESCRIPTOR_SET_LAYOUT_BINDING_FLAGS_CREATE_INFO_EXT,
     };
 
-    VkDescriptorBindingFlagsEXT bindFlags[3] = {
-        VK_DESCRIPTOR_BINDING_PARTIALLY_BOUND_BIT_EXT,
-        0,
+    VkDescriptorBindingFlagsEXT bindFlags[4] = {
+        VK_DESCRIPTOR_BINDING_PARTIALLY_BOUND_BIT_EXT, 0,
         VK_DESCRIPTOR_BINDING_PARTIALLY_BOUND_BIT_EXT,
-    };
+        VK_DESCRIPTOR_BINDING_VARIABLE_DESCRIPTOR_COUNT_BIT_EXT};
```

Comment:
Not required but if you add a `,` here after the last entry it formats the array a bit better with the closing brace on a new line.